drydock/templates/drydock/k8s/ingress/static-cache.yml

```diff
@@ -0,0 +1,33 @@
+{%- for service, config in DRYDOCK_NGINX_STATIC_CACHE_CONFIG.items() %}
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: ingress-static-{{ service }}
+  namespace: {{ K8S_NAMESPACE }}
+  annotations:
+    nginx.ingress.kubernetes.io/proxy-body-size: 8m
+    nginx.ingress.kubernetes.io/proxy-buffering: "on"
+    nginx.ingress.kubernetes.io/configuration-snippet: |
+      # Cache settings
+      proxy_cache_valid 404 10m;
+      proxy_cache_use_stale error timeout updating http_404 http_500 http_502 http_503 http_504;
+      proxy_cache static-cache;
+      proxy_cache_valid any 120m;
+      proxy_cache_bypass $http_x_purge;
+      add_header X-Cache-Status $upstream_cache_status;
+      {{ patch("static-cache-config") | indent(6)}}
+spec:
+  ingressClassName: nginx
+  rules:
+  - host: {{ config["host"] }}
```

Comment:
If we have 2 ingress objects pointing to the same host (let's say `my.host`), but only one ingress object defines the location `/static/`, when visiting https://my.host/static/, has the ingress defining the `/static/` location the priority to process the request?

Reply:
Yes, the ingress will end rendered on the nginx pod as:

```nginx
server {
  # Whatever the first ingress setups

  location /static/ {
     # our static settings
  }
}
```